src/hotspot/share/memory/metaspace.cpp

```diff
@@ -651,23 +651,44 @@ void Metaspace::ergo_initialize() {
   MaxMetaspaceSize = MAX2(MaxMetaspaceSize, commit_alignment());
 
   if (UseCompressedClassPointers) {
+    // Adjust size of the compressed class space.
+
+    const size_t res_align = reserve_alignment();
```

Comment:
Can you change the name to `root_chunk_size`?

Reply:
It feels wrong, since this is a deeply hidden implementation detail.\

I will remove this temporary variable, which will also make the diff smaller.

Reply:
Sounds OK, I wanted the name change to indicate that "hey, deep impl detail where we use this to mean something else".